test/runtests.jl

```diff
@@ -2830,6 +2831,22 @@ end
     ]
     @test isapprox(trans.rot, rot_real)
 
+    # Test rot isn't a reflection
+    cs_one = Float64[
+        1 -1  0  0  0  0
+        0  0  1 -1  0  0
+        0  0  0  0  1 -1
+    ]
+    cs_two = Float64[
+        -1  1  0  0  0  0
+         0  0  1 -1  0  0
+         0  0  0  0  1 -1
+    ]
+    trans = Transformation(cs_one, cs_two)
+    @test isapprox(trans.trans1, [0.0, 0.0, 0.0])
+    @test isapprox(trans.trans2, [0.0, 0.0, 0.0])
+    @test isapprox(det(trans.rot), 1.0)
```

Comment:
Could test for the actual rotation matrix?

Reply:
Sure, I made the y and z axis points further away to give them more weight, so the x axis reflection is ignored.